### ISSUE TITLE
JavaScript and TypeScript sorted differently

### ISSUE THREAD

**Issue by peterjuras** [renke/import-sort#10]
Hi!

I came across a bug, where the sorted result is different for JavaScript and TypeScript:

Version: **2.4.0**
Style: **module**

JavaScript Result:

```js
import { Time } from './time';
import { TimeZone } from './time-zone';
```

TypeScript Result:

```ts
import { TimeZone } from './time-zone';
import { Time } from './time';
```

Do you know why this happens? The JavaScript behavior is the correct one, but it seems to be different for TypeScript files

**Comment by renke** [comment]
That shouldn't happen. I am a bit short on time currently, but I will try to look into this.

**Comment by renke** [comment]
Hi @peterjuras 

this is probably related to #22 and #15 because you are using single quotes. This will be fixed in the next release!

**Comment by Alxandr** [comment]
I just ran into this myself. Will a fix be released soon?

**Comment by renke** [comment]
I hope I can release the new version this week, but I have to test it first with Atom (which I longer use personally, so it might take a while) and @peterjuras would have to upgrade [vsc-sort-imports](https://github.com/peterjuras/vsc-sort-imports) (thanks for you extension by the way!).

**Comment by renke** [comment]
The fix is now released with 3.0.
